jvb/src/main/kotlin/org/jitsi/videobridge/stats/callstats/CallstatsService.kt

```diff
@@ -86,23 +84,26 @@ class CallstatsService(
 
         statsManager?.let { statsManager ->
             logger.info("Subscribing to global stats with interval ${config.interval}")
-            statsTransport = CallStatsIOTransport(statsService).also { statsTransport ->
+            statsTransport = CallstatsTransport(statsService)
+                .also { statsTransport ->
```

Comment:
Was this newline on accident? The previous formatting looks like it made more sense

Reply:
yep, I don't know how that happened